bootstrap-coverage.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Can we move this file into a suitable sub folder? We already have a lot of files in the root folder which is already cluttering. We should not increase it if we can avoid it